### DISCUSSION TITLE
How to dynamicly  scale a clickhouse cluster or clickhouse-keeper cluster

### DISCUSSION THREAD

**Discussion by blylei** [ClickHouse/ClickHouse#50124] [Q&A] [answered]
For a beginer , I want to know how to scale a clickhouse cluster: add a replica to a running cluster or add a new clikhouse-keeper node

**Comment by tbragin** [comment, marked as answer]
For background material on ClickHouse sharding, replication, and scaling - some concepts in this tutorial can be helpful https://www.youtube.com/watch?v=vBjCJtw_Ei0
